js/overrides.js

```diff
@@ -1,61 +1,45 @@
-  },
-  'should send floating region by selector with vg': {skip: true},
-  'should send ignore region by selector with vg': {skip: true},
-  'should send ignore region by coordinates with css stitched': {skip: true},
-  'should send ignore region by coordinates with vg': {skip: true},
-  'should send floating region by coordinates with vg': {skip: true},
-  'check frame after manual switch to frame with css stitching classic': {skip: true},
-  'check frame after manual switch to frame with vg classic': {
-    config: {branchName: 'no-fully-by-default'},
-  },
-  'check regions by coordinates in overflowed frame with scroll stitching': {
-    config: {branchName: 'next'},
-  },
-  /**
-   *  Fails because of the difference between SCREEN SIZE and LAYOUT VIEWPORT SIZE.
-   *  We should use screen.width/screen.height for cropping/scaling and innerWidth/innerHeight to understand element location related to the viewport
-   */
-  'check window fully on android chrome emulator on desktop page': {skip: true},
-  'should send accessibility regions by selector with vg': {config: {branchName: 'v1'}},
-  'check region in frame hidden under top bar fully with css stitching': {skip: true},
-  'check region in frame hidden under top bar fully with scroll stitching': {skip: true},
-  'check region by selector in frame fully on firefox legacy': { skip: true },
-  // fails due to wrong calculation/rounding on other sdks
-  'appium android check window': {skipEmit: true},
-  'appium android check region with ignore region': {skipEmit: true},
-  // fails due to wrong screenshot size on other sdks
-  'should send region by selector in padded page': {skipEmit: true}
+    /// Updating baselines to JS
+    // Window
+    'check window two times with vg classic': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window with vg classic': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window fully and frame in frame fully with css stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully and frame in frame fully with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully and frame in frame fully with vg': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully with custom scroll root with css stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    // Region
+    'check region by coordinates in frame with css stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check region by coordinates in frame with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check overflowed region by coordinates with css stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check overflowed region by coordinates with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
    "check overflowed region by coordinates with scroll stitching": {config: {branchName: 'universal-sdk'}}, // was skipped
```